uint8[] agent_id
time start
duration duration
float32 travel_distance
duration cycle_duration
ChargeStatus charge_status
string name
